core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/Profiler.scala

```diff
@@ -124,6 +125,43 @@ class Profiler(hadoopConf: Configuration, appArgs: ProfileArgs, enablePB: Boolea
     progressBar.foreach(_.finishAll())
   }
 
+  def profileDriver(driverLogInfos: String): Unit = {
+    val profileOutputWriter = new ProfileOutputWriter(s"$outputDir/driver",
+      Profiler.DRIVER_LOG_NAME, numOutputRows, true)
+
+    try {
+      val unsupportedDrivers = processDriverLog(driverLogInfos)
+      profileOutputWriter.write(s"Unsupported operators in driver log",
+        unsupportedDrivers)
+    } finally {
+      profileOutputWriter.close()
+    }
+  }
+
+  def processDriverLog(driverlogPath: String): Seq[DriverLogUnsupportedOperators] = {
+    val source = Source.fromFile(driverlogPath)
+    // Create a map to store the counts for each operator and reason
+    var countsMap = Map[(String, String), Int]().withDefaultValue(0)
+    try {
+      // Process each line in the file
+      for (line <- source.getLines()) {
+        if (line.contains("cannot run on GPU") &&
```

Comment:
should protect against case sensitivity.  this is also very sensitive to us changing the wording here.  Can you check with Bobby on ideas to make this more resilient and to make sure this covers all the cases

Reply:
I think we should change the format for the future if we are planning on having tools about it.  Maybe something more official tags [NOT ON GPU]  that is easily parseable and unique.  Obviously will still have to keep old logic for anything up til that get implemented.

Reply:
Filed an issue in spark-rapids - https://github.com/NVIDIA/spark-rapids/issues/9728